timescaledb/getting-started/data-retention.md

```diff
@@ -1,79 +1,81 @@
 
 An intrinsic part of working with time-series data is that the relevance of
-highly granular data diminishes over time. New data is accumulated and old data
-is rarely, if ever, updated. It is therefore often desirable to delete old raw
+data can diminish over time. New data accumulates, and old data
+is rarely if ever, updated. Thus, it is often desirable to delete old raw
 data to save disk space.
 
-<highlight type="tip"> 
-In practice, old data is often downsampled first, such
-that a summary of it is retained (for example in a continuous aggregate), while
-the raw data points are then discarded via data retention policies. 
+In practice, continuous aggregates downsample the data and then
+ data retention policies discard historic raw data points.
+
+<highlight type="note">
+In the following image, dropping data on the underlying hypertable doesn't 
+affect the continuous aggregate. Your aggregate is unaffected as long as you do not refresh the continuous aggregate 
+for time periods where you dropped data.
 </highlight>
 
-Just like for continuous aggregates and compression, TimescaleDB provides an
-automated policy to drop data according to a schedule and defined rules.
-Automated data retention policies (along with compression and continuous
-aggregates) give you more control over the amount of data you retain and its
-granularity for specific time periods. These policies are "set it and forget it"
-in nature, meaning less hassle for maintenance and upkeep.
+  <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/continuous-aggregate-policy-retention.jpg" alt="Continuous aggregate with refresh and retention policies"/>
+
+So you have a continuous aggregate with a refresh policy, but you now may want to remove 
+old data in the underlying hypertable. There are two ways to drop data from the 
+underlaying hypertable: 
+1. Automatic data retention policy
+2. Manually dropping chunks
+
+In this first section, we look at the automatic refresh policy and remove data 
+on a schedule. 
+
+## Automatic data retention policies
 
-For example, here is a data retention policy which drops chunks consisting of
-data older than 25 years from the hypertable `weather_metrics`:
+**Automated data retention policies** drop data according to a schedule and defined rules. 
+These policies are "set it and forget it" in nature, meaning less hassle 
+for maintenance and upkeep.
+
+For example, you may want to continually remove stock data in the underlying hypertable 
+`stocks_real_time` for time periods greater than three months ago. You can use this code with 
+the [`add_retention_policy()`][retention-policy] function to set up the automatic retention policy for this:
 
 ```sql
--- Data retention policy
--- Drop data older than 25 years
-SELECT add_retention_policy('weather_metrics', INTERVAL '25 years');
+SELECT add_retention_policy('stocks_real_time', INTERVAL '3 months');
```

Comment:
Same as the compression comment. Maybe we give more data, but I think we should write this with the assumption that they'll see actual stuff happen. So, if it's a month of data, then we can do data retention after 4 weeks and then maybe the compression after 2?